src/test/java/com/philips/research/bombar/core/domain/DependencyTest.java

```diff
@@ -138,14 +138,47 @@ void addsRelations() {
     @Test
     void addsUsages() {
         final var target = new Dependency(ID, "Parent");
-
         dependency.addUsage(target);
 
         final var dependencies = dependency.getUsages();
 
         assertThat(dependencies).containsExactly(target);
     }
 
+    @Test
+    void noStrongUsageForOrphan() {
+        assertThat(dependency.getStrongestUsage()).isEmpty();
+    }
+
+    @Test
+    void findsStrongestRelationShipFromOwnUsage() {
+        final var parent = new Dependency(ID, "Parent");
+        final var firstChild = new Dependency(ID, "First");
```

Comment:
Isn't this about a "child" and its "sibling"? (That at least indicates the primary subject for this test.)